docs/DE0003.md

```diff
@@ -14,4 +14,5 @@ T:System.Net.HttpWebRequest
 
 ## Recommendation
 
-Use `HttpClient` instead.
+For HttpWebRequest: use `HttpClient` instead.
+For FtpWebRequest: search for third party FTP client.
```

Comment:
Recommending a specific FTP client library that we know targets netstandard would be really nice.

Reply:
@halter73 do you have a specific recommendation beside the SO link above?

Reply:
Nope. Sorry. That's why I was hoping for one.